src/md/release.md

```diff
@@ -150,6 +150,47 @@ mkdir svn-dist && cp *-source-* svn-dist/
 svn import svn-dist https://dist.apache.org/repos/dist/dev/apex/${RNAME}-RC1 -m "Apache Apex v${rv}-RC1"
 ```
 
+## Build and Deploy Documentation
+
+The documentation will be generated as static HTML files and copied into the ```apex-site``` repository. It will be available at an alternate URL location until the release is promoted, at which time it will also be promoted to the main website location.
+
+Do the following setup setups before building and deploying the documentation. Clone the apex-site repository into a folder called ```apex-site``` at the same level as the current repository. Set the following environment variables
+
+For -core releases:
+
+```
+REPO_NAME=apex-core
+DOC_NAME=apex
+```
+
+For -malhar releases:
+
+```
+REPO_NAME=apex-malhar
+DOC_NAME=malhar
+```
+
+Execute the following commands to build and deploy the documentation. **Note**: Until [mkdocs #859](https://github.com/mkdocs/mkdocs/issues/859) is resolved and available for download, use mkdocs built against [master](https://github.com/mkdocs/mkdocs).
+
+```bash
+# build docs, they would be generated in the site folder
+mkdocs build --clean
+
+# copy docs from site into target folder on apex-site
+cd ../apex-site
+git checkout asf-site
+rm -rf docs/${DOC_NAME}-${rv}
```

Comment:
I like the idea of not adding -RC, and just using the version directly.  However, I can see the argument can be made that 3.4.0 and 3.4.1 can have security or other feature breaking changes, and we potentially should support independent docs for both.  In light of this, I suggest we use `X.Y.Z` for docs versioning.  I know this means extra space used by patch versions, and extra changes to docs when a patch is released, but gives us flexibility to support any documentation change scenario.  So we will have:

```
docs/apex/3.3.0
docs/apex/3.4.0
docs/apex/3.4.1
docs/apex -> docs/apex/3.4.1
```